### ISSUE TITLE
The v2.3.5 release binaries do not seem to include the update to minio-go

### ISSUE THREAD

**Issue by justfalter** [buchgr/bazel-remote#533]
@mostynb  I'm testing out the [bazel-remote-2.3.5-linux-x86_64](https://github.com/buchgr/bazel-remote/releases/download/v2.3.5/bazel-remote-2.3.5-linux-x86_64), and it is behaving as though it does not have the update to minio-go from #531. I get the 120s delay in AWS land.

Taking a look at the strings for the binary, it looks like it's got `minio-go/v7.0.20`, but should have `minio-go/v7.0.24`

```
$ strings bazel-remote-2.3.5-linux-x86_64 | grep -o 'minio-go/v7\.0...............'
minio-go/v7.0.20Number of by
minio-go/v7.0.20 Stack trace
```

When I do the same thing on a linux binary built from master, I get `minio-go/v7.0.24`, and things behave as I would expect:

```
$ strings bazel-remote | grep -o 'minio-go/v7\.0...............'
minio-go/v7.0.24Number of by
minio-go/v7.0.24 Stack trace
```

**Comment by mostynb** [comment]
Checking...

**Comment by mostynb** [comment]
Ahh, I know the problem. The bazel build is out of sync with go.mod - I'll fix that and make a new release. Thanks for checking! I'll need to find a way to add a test for this later.

**Comment by mostynb** [comment]
Fixed in v2.3.6, and added a CI test to catch this.
